tests/EMBEDDED_PYTHON.DATA

```diff
@@ -0,0 +1,70 @@
+RUNSPEC
+
+DIMENS
+   10 10 3 /
+
+
+GRID
+
+DX
+   	300*1000 /
+DY
+	300*1000 /
+DZ
+	100*20 100*30 100*50 /
+
+TOPS
+	100*8325 /
+
+PORO
+   	300*0.3 /
+
+PERMX
+   300*1 /
+
+PERMY
+   300*1 /
+
+PERMZ
+   300*1 /
+
+SCHEDULE
+
+PYACTION
+  WCLOSE UNLIMITED /
+  'wclose.py' /
```

Comment:
Do we no longer need `PYEND`?

Reply:
This is described in some detail in the #1591 PR description; point is that the Python code is relegated to a separate file and then we no longer need `PYEND`; it just became too complex to handle both the metadata part:
```
WCLOSE UNLIMITED /
```
and the literal Python code in the same keyword in the `.DATA` file. I don't know if it is an improvement or - I guess it depends on the eyes of the beholder.